# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/dns/TRR.cpp
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/dns/TRR.cpp
--- a/netwerk/dns/TRR.cpp	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/dns/TRR.cpp	Tue Feb 12 16:08:25 2019 +0000
@@ -307,7 +307,7 @@
           NS_LITERAL_CSTRING("application/dns-message")))) {
     LOG(("TRR::SendHTTPRequest: couldn't set content-type!\n"));
   }
-  if (NS_SUCCEEDED(httpChannel->AsyncOpen2(this))) {
+  if (NS_SUCCEEDED(httpChannel->AsyncOpen(this))) {
     NS_NewTimerWithCallback(getter_AddRefs(mTimeout), this,
                             gTRRService->GetRequestTimeout(),
                             nsITimer::TYPE_ONE_SHOT);
@@ -448,7 +448,7 @@
     return rv;
   }
 
-  rv = pushed->AsyncOpen2(this);
+  rv = pushed->AsyncOpen(this);
   if (NS_FAILED(rv)) {
     return rv;
   }